core/pkg/monitor/gpu_apple.go

```diff
@@ -54,12 +54,8 @@ func (g *GPUApple) parseStats() (map[string]interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
-	stats := make(map[string]interface{})
-	err = json.Unmarshal(rawStats, &stats)
-	if err != nil {
-		return nil, err
-	}
-	return stats, nil
+
+	return simplejsonext.UnmarshalObject(rawStats)
```

Comment:
can we use default json here?

Reply:
oh, good point, this should be a standard JSON string without inf or nan

Reply:
done